src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -320,4 +320,6 @@ object ScalafixPlugin extends AutoPlugin {
       }
     }
   }
+
+  final class UninitializedError extends RuntimeException("uninitialized value")
```

Comment:
What is this used for?

Reply:
ah, `UninitalizedError` was deprecated in `2.12.7` so I added it here. It's used here https://github.com/scalacenter/sbt-scalafix/blob/master/src/main/scala/scalafix/sbt/ScalafixPlugin.scala#L101

Reply:
I see, makes sense!